src/Theme/ClassicTheme.php

```diff
@@ -46,15 +47,21 @@ public function onTestFinished(TestResult $result): void
             $throwable = $throwable->previous();
         }
 
-        if ($result->trace && !$result->trace->suppressed) {
-            printf(
-                Color::colorize("fg-$statusColour", '%s%s: %s in %s on line %s%1$s%1$s'),
-                PHP_EOL,
-                $result->status->name,
-                $result->trace->message,
-                $result->trace->file,
-                $result->trace->line,
-            );
+        $firstIssue = true;
```

Comment:
[I still don't think we need this flag](https://3v4l.org/hpg4j).

Reply:
The traces array collects all events, suppressed or not, so the first element may not be displayed at all. To avoid the flag I'd need to either change the existing behaviour and only collect non-suppressed, or filter out suppressed issues before the loop, or something worse.